### PR TITLE
impact grenades are back

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Readds impact grenades.
Adds directional sprites for them when activated.
image

## Why It's Good For The Game
Impact grenades were removed because of the fact that you could stunlock people with them, considering the fact that stunning on explosions is gone, I don't see why they shouldn't come back as another grenade type.

## Changelog
:cl:
add: Added back impact grenades to marine vendors
/:cl:

### CODE REVIEW

**Comment by KrabSpider** [comment]
Man, I am **so** excited to be on the receiving end of a never ending barraging of stagger-slows from a some Marine with a T-70! 
...Literally my only concern.

**Comment by ilikesaying** [comment]
**ohno**

**Comment by KornFlaks** [comment]
300 round start impact grenades. I can see nothing wrong with this. Absolutely nothing at all.

**Comment by RenGusta** [comment]
I'm interested to see if it's actually good compared to HEDP or whatever, mostly if the radius is smaller it'll immediately be harder to capitalise on, but easier to initially utilise

**Comment by SilencedMP5A5** [comment]
wait, HOW MANY DO WE GET AT ROUNDSTART??

mjp what are you doing you fool

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -272,6 +272,7 @@
 		),
 		"Grenades" = list(
 			/obj/item/explosive/grenade/frag = 600,
+			/obj/item/explosive/grenade/impact = 300,
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:275]:
```suggestion
			/obj/item/explosive/grenade/frag = 600,
			/obj/item/explosive/grenade/impact = 50,
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/vending/marine_vending.dm (hunk 1 above)
Comment (Pariah919) [inline code/game/objects/machinery/vending/marine_vending.dm:275]:
```suggestion
			/obj/item/explosive/grenade/impact = 125,
```

**Comment by Pariah919** [comment]
reduced it to 125

File: code/game/objects/machinery/vending/marine_vending.dm (hunk 1 above)
Comment (SplinterGP) [reply code/game/objects/machinery/vending/marine_vending.dm]: i don't see why not make them alot, theyre similar to HEDP's the only difference is they blow on direct hits on living things when used on GL's, but they do same damage

**Review by TiviPlus — APPROVED** [review]

File: code/game/objects/machinery/vending/marine_vending.dm (hunk 1 above)
Comment (InterroLouis) [reply code/game/objects/machinery/vending/marine_vending.dm]: They actually have lower radius than regular grenades when thrown, 3 tiles compared to 4, and single tile light explosion if it hits something along with 2 tiles flash radius.

**Comment by turnop** [comment]
>buffs the most broken item in the game

**Pull request merged**